bcp/client/src/webpack/helper/toGeoJSON.js

```diff
@@ -252,5 +253,64 @@ export const fetchAsset = async (id, is_equipment) => {
         }
     }
 };
+export const fetchAssetList = async (assetMode) => {
 
+    const aircraftData = await fetch('/data/aircraft.json');
+    const equipmentData = await fetch('/data/equipment.json');
+    const equipmentJSON = await equipmentData.json();
+    const aircraftJSON = await aircraftData.json();
+    const availabilityData = await fetch('/data/availability.json');
+    const availabilityJSON = await availabilityData.json();
+    const {aircraft} = aircraftJSON;
+    const {equipment} = equipmentJSON;
+    const currentLocationsData = await fetch('/data/currentLocations.json');
+    const currentLocationsJSON = await currentLocationsData.json();
```

Comment:
I've no idea if this will lead to any performance issues, but is there a need to load this data sequentially rather than concurrently?